R/caretStack.R

```diff
@@ -229,6 +338,22 @@ check_caretStack <- function(object) {
   )
 }
 
+#' @title Check original_features_included and newdata
+#' @description Make sure newdata contains all the original features used in training
+#'
+#' @param newdata a data.frame of new data
+#' @param original_features_included a character vector of original features used in training
+#' @keywords internal
+check_newdata_features <- function(newdata, original_features_included) {
```

Comment:
Lets use one checking function for BOTH training and predictions.

In both cases you have newdata (newX at training) and original_features_included

raise an error if newdata is NULL

raise an error if original_features_included is not a character vector

raise an error if any original_features_included are not in newX/newdata

Reply:
Ok, I will make this change too.